README.md

```diff
@@ -7,125 +7,39 @@ Nextcloud app to sign PDF documents.
 <img src="img/LibreSign.png" />
 
 **Table of contents**
-- [Releases compatibility](#releases-compatibility)
 - [Setup](#setup)
-  - [Java and JSignPDF](#java-and-jsignpdf)
-    - [Standalone](#standalone)
-    - [Using Docker](#using-docker)
-  - [CFSSL](#cfssl)
-    - [CFSS server handmade install](#cfss-server-handmade-install)
-    - [With docker-compose](#with-docker-compose)
+  - [Standalone](#standalone)
   - [Admin settings](#admin-settings)
-  - [Validation page](#validation-page)
 - [Integrations](#integrations)
 - [Full documentation](#full-documentation)
 - [Contributing](#contributing)
 
-## Releases compatibility
-
-| LibreSign | Nextcloud | JSignPDF |
-| --------- | --------- | -------- |
-| 6.x       | 25        | 2.1.0    |
-| 5.x       | 24        | 2.1.0    |
-| 4.x       | 23        | 2.1.0    |
-| 3.x       | 22        | 2.1.0    |
-| 2         | 22        | 1.6.5    |
-
 ## Setup
 
-### Java and JSignPDF
+### Standalone
+Help to usage in configure cfssl command:
```

Comment:
Não consegui entender essa frase

Reply:
Are tips to use the command to configure cfssl.

Reply:
So, I think could be written like this: Helps in configuring cfssl command: